apps/cricket-score-frontend/src/components/match/ErrorDisplay.tsx

```
@@ -0,0 +1,39 @@
+  onRetry: () => void;
+}
+
+export const ErrorDisplay: React.FC<ErrorDisplayProps> = ({ error, onRetry }) => (
+  <div className="flex justify-center items-center min-h-screen bg-gradient-theme p-4">
+    <motion.div
+      initial={{ opacity: 0, scale: 0.9 }}
+      animate={{ opacity: 1, scale: 1 }}
+      className="w-full max-w-md text-center glass-card border border-red-500/40 p-8"
+    >
+      <div className="w-16 h-16 mx-auto bg-red-500/30 backdrop-blur-sm rounded-full flex items-center justify-center border-2 border-red-500/50 relative shadow-xl">
+        <svg xmlns="http://www.w3.org/2000/svg" className="h-8 w-8 text-red-400" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)